packages/build/tests/helpers/normalize.js

```diff
@@ -125,6 +124,8 @@ const NORMALIZE_REGEXPS = [
   // npm install logs look different on Node 8.3.0
   [/\snpm ERR! 404([^]*npm ERR! 404)?.*/g, ''],
   [/No valid versions/g, 'No versions'],
+  // npm error logs are shown differently in CI
+  [/\s*npm ERR!\s+\/external\/path/g, ''],
```

Comment:
This is a log line that just shows where the npm error log is located. For some reason, it seems to appear locally on my machine, but not in CI. So we need to normalize it.